tests/test_field.py

```diff
@@ -0,0 +1,161 @@
+import os.path as path
+
+import unittest
+from math import *
+
+import traceon.mesher as M
+from traceon.geometry import *
+import traceon.excitation as E
+import traceon.solver as S
+from traceon.field import *
+
```

Comment:
Consider adding the following to get less noise in the terminal when running tests there

```Python
import traceon.logging as logging

logging.set_log_level(logging.LogLevel.SILENT)
```